bazel/genrule_repository.bzl

```diff
@@ -74,6 +74,13 @@ genrule_cc_deps = rule(
     implementation = _genrule_cc_deps,
 )
 
+def _absolute_bin(path):
+  # If the binary path looks like it's relative to the current directory,
+  # transform it to be absolute by appending "${PWD}".
+  if "/" in path and not path.startswith("/"):
+    return '"${PWD}"/%r' % (path,)
```

Comment:
This seems very hacky, but is fine to unblock things. Please file an issue with Bazel on the compiler autoconfigure (we want absolute paths) and maybe add to this comment a tracking TODO.

Reply:
Filed https://github.com/bazelbuild/bazel/issues/3910